code/services/FacebookCount.php

```diff
@@ -9,19 +9,17 @@ class FacebookCount extends Controller implements SocialServiceInterface
 {
 
     public $service = 'Facebook';
+
     public $statistic = array(
         'share_count',
         'like_count',
         'comment_count'
     );
     public $requestCount = 5;
 
-    private function getFacebookCall($urls)
+    private function getFacebookCall($url)
     {
-        return 'https://api.facebook.com/method/fql.query' .
-            '?query=select%20url,share_count,like_count,comment_count%20' .
-            'from%20link_stat%20where%20url%20in("'
-            . urlencode(implode('","', $urls)).'")';
+        return "http://graph.facebook.com/?id={$url}";
```

Comment:
One of the core requirements, when this module was first scoped, was to ensure we're hitting APIs as little as possible.  We need to check upwards of 24000 URLs per day, so we need to make sure we're utilizing API quotas efficiently.